buildSrc/src/main/groovy/nva.cristin.service.core.java-conventions.gradle

```diff
@@ -16,67 +15,32 @@ dependencies {
     implementation libs.slf4j.api
 }
 
-sourceCompatibility = JavaVersion.VERSION_11  // source-code version and must be <= targetCompatibility
-targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
-
 repositories {
     mavenCentral()
     maven { url 'https://jitpack.io' }
 }
 
-compileJava.options.encoding = 'utf-8'
-
-tasks.withType(JavaCompile) {
-    options.encoding = 'utf-8'
-}
-
-tasks.named('test') {
+tasks.named('test', Test) {
     useJUnitPlatform()
-    failFast = true
     testLogging {
         events 'skipped', 'passed', 'failed'
     }
-    finalizedBy jacocoTestReport
-}
-
-test {
     environment "ALLOWED_ORIGIN", "*"
     environment "CRISTIN_API_URL", "https://api.cristin-test.uio.no/v2"
     environment "BASE_PATH", "cristin"
     environment "DOMAIN_NAME", "api.dev.nva.aws.unit.no"
+    finalizedBy jacocoTestReport
 }
 
-task karateTest(type: Test) {
-    useJUnitPlatform {
-        includeTags('karateTest')
-    }
-
-    environment "ALLOWED_ORIGIN", "*"
-    environment "CRISTIN_API_URL", "https://api.cristin-test.uio.no/v2"
-    environment "DOMAIN_NAME", "api.dev.nva.aws.unit.no"
-
-    systemProperty "karate.options", System.properties.getProperty("karate.options")
-    systemProperty "karate.env", System.properties.getProperty("karate.env")
-    outputs.upToDateWhen { false }
-}
-
-task createTestUsers(type: Test) {
-    useJUnitPlatform {
-        includeTags('createTestUsers')
-    }
-
-    outputs.upToDateWhen { false }
-}
-
-task deleteTestUsers(type: Test) {
-    useJUnitPlatform {
-        includeTags('deleteTestUsers')
+sourceSets {
+    test {
+        resources {
+            srcDir file('src/test/java')
+            exclude '**/*.java'
+        }
     }
-
-    outputs.upToDateWhen { false }
 }
 
-
 pmd {
     toolVersion = '6.20.0'
```

Comment:
there is a more recent version, but i dont know if it is necessary to use the latest

Reply:
testet toolversion 7.0.rc3, couldn't spot any difference

Reply:
ask @anetteOlli  about which version to use, maybe its ok with 6.20